package.json

```diff
@@ -124,9 +124,9 @@
     "@types/webpack-sources": "3.2.0",
     "@typescript-eslint/eslint-plugin": "6.6.0",
     "@typescript-eslint/parser": "6.6.0",
-    "@vitest/coverage-c8": "0.32.4",
-    "@vitest/coverage-istanbul": "~0.32.0",
-    "@vitest/ui": "~0.32.0",
+    "@vitest/coverage-c8": "0.33.0",
+    "@vitest/coverage-istanbul": "~0.34.0",
+    "@vitest/ui": "~0.34.0",
```

Comment:
The version of '@vitest/coverage-istanbul' has been upgraded. Make sure to test the new version thoroughly to ensure it doesn't break existing functionality.